locale/justoverclock-flarum-ext-realtimecode

```diff
@@ -0,0 +1,5 @@
+  forum:
```

Comment:
First line must be the original first line (file name must be first line of original file + '.yml' suffix.
In this case flarum-ext-realtimecode.yml.